src/server/scan/listener/helpers/process-scan-complete.js

```diff
@@ -1,36 +1,36 @@
+import { config } from '~/src/config'
 import {
   isReady,
   isUploadPending,
   uploadStatus
 } from '~/src/server/common/helpers/upload-status'
+import { fileStatus } from '~/src/server/common/constants/file-status'
 import { sendSqsMessage } from '~/src/server/common/helpers/sqs/send-sqs-message'
-import { isInfected } from '~/src/server/common/helpers/file-status'
-import { config } from '~/src/config'
 import { createUploadLogger } from '~/src/server/common/helpers/logging/logger'
 
 const callbackQueueUrl = config.get('sqsScanResultsCallback')
-async function processScanComplete(server, uploadId, fileId) {
-  const { files, uploadDetails } =
-    await server.redis.findUploadAndFiles(uploadId)
 
+async function processScanComplete(server, uploadId) {
+  const uploadAndFiles = await server.redis.findUploadAndFiles(uploadId)
+  const files = uploadAndFiles?.files
+  const uploadDetails = uploadAndFiles?.uploadDetails
   const uploadLogger = createUploadLogger(server.logger, uploadDetails)
   const isPending = isUploadPending(uploadDetails.uploadStatus)
 
   if (isPending && isScanningComplete(files)) {
     uploadDetails.uploadStatus = uploadStatus.ready.description
-    uploadDetails.ready = new Date()
-    uploadDetails.numberOfInfectedFiles = numberOfInfectedFiles(files)
-    await server.redis.storeUploadDetails(uploadId, uploadDetails)
+    uploadDetails.ready = new Date().toISOString()
+    uploadDetails.numberOfRejectedFiles = numberOfRejectedFiles(files)
 
-    const readyUploadLogger = createUploadLogger(server.logger, uploadDetails)
-    readyUploadLogger.info(`Upload marked as ready`)
+    await server.redis.storeUploadDetails(uploadId, uploadDetails)
+    server.logger.info({ uploadDetails }, 'Upload marked as ready')
 
     if (uploadDetails.scanResultCallbackUrl) {
       try {
         await sendSqsMessage(server.sqs, callbackQueueUrl, { uploadId })
       } catch (error) {
-        readyUploadLogger.error(
```

Comment:
It's quite useful knowing what file request came in when the `processScanComplete` request was being handled imo

Reply:
I've removed the child logger. There were multiple child loggers added due to the object being logged being mutated. ChildLoggers are useful if you wish to log the same thing in multiple places. If the thing you are logging is mutated then just use a standard logger. To log items in the first arg in pino loggers then need to be nested objects

see `mergingObjects` - https://github.com/pinojs/pino/blob/main/docs/api.md#loggerinfomergingobject-message-interpolationvalues

Reply:
Will revert/refactor to what was there previously